admin/views/tabs/tool/wpseo-import.php

```diff
@@ -10,17 +10,17 @@
 }
 
 ?>
-<p><?php _e( 'Import settings by locating <em>settings.zip</em> and clicking "Import settings"', 'wordpress-seo' ); ?></p>
+<p><?php esc_html_e( 'Import settings by locating <em>settings.zip</em> and clicking "Import settings"', 'wordpress-seo' ); ?></p>
```

Comment:
This went wrong, the `<em>` tags are escaped
![tools_-_yoast_seo_ _local_wordpress_dev_ _wordpress](https://user-images.githubusercontent.com/325040/32219450-897e2ba6-be2e-11e7-927e-ccb3f5d2c89d.png)

Reply:
@andizer Thanks for finding this! I'd looked the PR over two, three times, but still missed this one. Glad you didn't!